Description: Fix FTBS with gcc5
  Swiching to gcc5 cases of potentially used unititialized variables
  cause build failures. In this case it a false positive but hard to
  spot for the compiler (after shifting in 8 bits the variables are
  initialized). But working around that by initializing both to 0.
Forwarded: not yet
Author: Stefan Bader <stefan.bader@canonical.com>
Last-Update: 2015-08-12

Index: dpdk/lib/librte_pmd_ixgbe/ixgbe/ixgbe_phy.c
===================================================================
--- dpdk.orig/lib/librte_pmd_ixgbe/ixgbe/ixgbe_phy.c	2015-08-10 13:57:49.567096563 +0200
+++ dpdk/lib/librte_pmd_ixgbe/ixgbe/ixgbe_phy.c	2015-08-12 11:19:56.869898129 +0200
@@ -116,8 +116,8 @@ STATIC s32 ixgbe_read_i2c_combined_gener
 	int max_retry = 10;
 	int retry = 0;
 	u8 csum_byte;
-	u8 high_bits;
-	u8 low_bits;
+	u8 high_bits = 0;
+	u8 low_bits = 0;
 	u8 reg_high;
 	u8 csum;
 
